src/main/Main.js

```diff
@@ -259,13 +251,16 @@ async function discordEarnUpdate({ results, harvests }) {
         const harvest = harvests[i];
         if (!harvest.earnDecision) continue;
         if (value) {
-            const msg = [];
-            msg.push('```');
-            msg.push(`Snowglobe:   ${harvest.name}`);
-            msg.push(`Swept:       ${Util.displayBNasFloat(harvest.available, 18, 5)} ${harvest.wantSymbol} ($${Util.displayBNasFloat(harvest.availableUSD, 18)})`);
-            msg.push(`Transaction: ${value?.transactionHash ?? '[real tx hash]'}`);
-            msg.push('```');
-            DiscordBot.sendMessage(msg.join("\n"), CONFIG.DISCORD.HARVESTS);
+            const embedObj = {
+                Color:'0x00aaff',
+                Title:`Snowglobe: ${harvest.name}`,
+                Thumbnail:Util.thumbnailLink(harvest.name),
+                URL:Util.cchainTransactionLink(value.transactionHash),

```

Comment:
extra comma

Reply:
trailing commas have always been acceptable in arrays, and in objects since ECMAScript 5